app/models/achievement.rb

```diff
@@ -5,4 +5,9 @@ class Achievement < ActiveRecord::Base
   validates :title, :side, :points, presence: true
 
   enum side: %w(corp runner)
+
+  def to_s
+    "#{self.class.name} [#{self.side}/#{self.points}] '#{self.title}' / '#{self.description}'"
```

Comment:
Hehe, my Java style coming through. I'll remove the redundant `self` and break up the line for length.